recipes/sshpubkeys/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "sshpubkeys" %}
+{% set version = "3.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "b388399caeeccdc145f06fd0d2665eeecc545385c60b55c282a15a022215af80"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Could it be ``noarch: python`` ?
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
  noarch: python
```